pages/app/orgsettings.tsx

```diff
@@ -0,0 +1,103 @@
+import Head from 'next/head'
+import React, { useState } from 'react'
+import { ArrowNarrowLeftIcon, UserCircleIcon } from '@heroicons/react/outline'
+import NavBar from '../../components/NavBar'
+import { useRouter } from 'next/router'
+
+export default function OrgSettings() {
+  const router = useRouter()
+  const [DeleteModal, setDeleteModal] = useState(false)
+  const orgName = 'My Org'
+  const orgMembers = ['ExampleUser@gmail.com', 'JohnDoe@gmail.com', 'JaneDoe@gmail.com']
+  return (
+    <>
+      <Head>
+        <title>Organization Settings</title>
+      </Head>
+      <NavBar />
+      {DeleteModal && (
+        <div>
+          <div className="fixed h-full w-full top-0 bg-black opacity-70 z-10"></div>
+          <div className=" absolute w-screen flex flex-col items-center justify-center h-screen z-20">
+            <div className=" bg-white opacity-100 border rounded-2xl flex flex-col items-center justify-center left-1/4 max-w-lg ">
+              <label className="p-8 font-semibold text-2xl">Delete this organization?</label>
+              <label className="pb-6 text-lg font">
+                Deleting your organization is irreversible.
+              </label>
+              <div className="flex flex-row  w-imgSM justify-evenly pb-10 ">
+                <button
+                  className="py-2 px-4 w-44 bg-gray-400 text-white rounded-lg hover:bg-gray-600 cursor-pointer transition duration-500 ease-in-out text-sm"
+                  onClick={() => setDeleteModal(false)}
+                >
+                  Cancel
+                </button>
+                <button
+                  className="py-2 px-4 w-44 bg-red-500 text-white rounded-lg hover:bg-red-600 cursor-pointer transition duration-500 ease-in-out text-sm"
+                  onClick={() => ''}
+                >
+                  Delete Organization
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      )}
+      <div className="max-w-6xl container mx-auto flex flex-col items-center justify-center relative">
+        <div className="w-3/4 flex flex-row justify-center items-center mb-24 relative">
+          <div className="w-1/5 mt-1">
+            <button>
+              <ArrowNarrowLeftIcon className=" h-6 w-6" />
+            </button>
+          </div>
+          <label className=" w-4/5 font-semibold text-4xl">{orgName} Settings</label>
+        </div>
+        <div className="max-w-lg relative">
+          <div>
+            <label className="text-lg text-gray-500">Organization Name</label>
+            <input
+              className="rounded-lg h-10 border-gray-300 border block mb-6 mt-2 px-28"
+              value=""
```

Comment:
We want to make sure that people can edit this input.
```suggestion
              type="text"
              placeholder={orgName}
```